src/Modules.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   'grid-overlay': require('./modules/GridOverlay'),
   'import-image': require('./modules/ImportImage'),
   'minify-image': require('./modules/MinifyImage'),
-  'invert': require('image-sequencer-invert'),
+  // 'invert': require('image-sequencer-invert'),
```

Comment:
This had been kept as an example of a module literally living outside the project, which is nice. Maybe we can add a comment back in here to explain this, in follow-up.